packages/react/src/host/RemoteFragment.tsx

```diff
@@ -0,0 +1,38 @@
+import {memo, useEffect, useMemo} from 'react';
+import {
+  RemoteFragmentSerialization,
+  createRemoteReceiver,
+  ACTION_MOUNT,
+} from '@remote-ui/core';
+import type {RemoteReceiver} from '@remote-ui/core';
+
+import type {Controller} from './controller';
+// eslint-disable-next-line import/no-cycle
```

Comment:
We have cycle dependency issue when using sub receiver approach. I don't see how we can get around it or use different approach.